docs/git-comparison.md

```diff
@@ -243,17 +249,17 @@ parent.
       <td><code>git stash</code></td>
     </tr>
     <tr>
-      <td>Start working on a new change based on the &lt;main&gt; bookmark</td>
+      <td>Start working on a new change based on the &lt;main&gt; bookmark/branch</td>
       <td><code>jj new main</code></td>
       <td><code>git switch -c topic main</code> or
         <code>git checkout -b topic main</code> (may need to stash or commit
         first)</td>
     </tr>
     <tr>
-      <td>Move bookmark A onto bookmark B</td>
+      <td>Move bookmark/branch A onto bookmark/branch B</td>
       <td><code>jj rebase -b A -d B</code></td>
       <td><code>git rebase B A</code>
```

Comment:
@martinvonz: Would it be correct to say `git rebase B A --update-refs` here and remove the comment below? Or would you prefer to mention `--update-refs` in the comment? I'd probably put it into a separate commit.

Reply:
No, I don't think that would be correct because it would still not rebase descendants. If you had a linear chain of bookmarks A..C, then `git rebase D C` would rebase A, B, and C. But it would still fail if it was not linear, so I think it makes sense to leave the comment here.

Reply:
OK, I'll leave it be then. For some strange reason, I stopped following new Git features a few years ago :)